### ISSUE TITLE
nodejs version is outdated

### ISSUE THREAD

**Issue by dantuluridilip** [bmalnad/s3-pgp-encryptor#6]
the code is not working as nodejs10.x is outdated.
it has to be updated to the latest version (nodejs14.x)

**Comment by MSanteler** [comment]
Looks like node was bumped, but aws is still using the old version of the application and fails... need to update zip on s3?

**Comment by bmalnad** [comment]
Yeah, I'll try to get to it today.  In the meantime, feel free to clone the
repo and deploy to your own account.

Dan
